assets/xml/objects/object_anubice.xml

```diff
@@ -0,0 +1,20 @@
+<Root>
+    <File Name="object_anubice" Segment="6">
+        <Skeleton Name="gAnubiceSkel" Type="Normal" LimbType="Standard" Offset="0x3990"/>
+
+        <Animation Name="gAnubiceFallDownAnim" Offset="0x348"/>
+        <Animation Name="gAnubiceAttackingAnim" Offset="0x78C"/>
+        <Animation Name="gAnubiceIdleAnim" Offset="0xF74"/>
+        <DList Name="gAnubiceEyesDL" Offset="0x3468"/>
+        <DList Name="gAnubiceFireAttackDL" Offset="0x3510"/>
+
+        <!-- The following are unused -->
+        <Animation Name="gAnubiceLaughingAnim" Offset="0x144"/>
+        <Animation Name="gAnubiceStandUpAnim" Offset="0x930"/>
+        <DList Name="gAnubiceUnusedParticleDL1" Offset="0x3610"/>
+        <DList Name="gAnubiceUnusedParticleDL2" Offset="0x3688"/>
```

Comment:
we want the suffixes to be the last thing, so particle1DL and particle2DL
we can probably name them after their purpose tho. iirc one dlist has the texture and the other has the geometry?